VideoLocker/src/main/java/org/edx/mobile/view/WebViewFindCoursesActivity.java

```diff
@@ -69,9 +76,54 @@ public boolean onQueryTextChange(String newText) {
                 return false;
             }
         });
+        searchView.setOnQueryTextFocusChangeListener(new View.OnFocusChangeListener() {
+            @Override
+            public void onFocusChange(View view, boolean queryTextFocused) {
+                if (!queryTextFocused) {
+                    searchActive = false;
+                    searchItem.collapseActionView();
+                    dismissKeyboard();
+                    showDrawerButton(true);
+                }  else {
+                    searchActive = true;
+                    showDrawerButton(false);
```

Comment:
This could be called just once outside the if/else blocks, i.e. `showDrawerButton(!queryTextFocused)`.